content/events/16-17/LAN.md

```diff
@@ -9,7 +10,18 @@ locationlink: Therminal,Ghent Belgium
 description: Zeus' vijfde LAN-Party
 ---
 
-De vijfde editie van Zeus WPI's LAN-Party komt eraan!
-Hou alvast de datum vrij!
+# It's that time of year again
 
-Wij houden jullie op de hoogte!
+Je leest het goed, op **4 november** vindt onze vijfde editie van onze jaarlijkse LAN-Party plaats in **[Studentenhuis De Therminal](http://student.ugent.be/)** (Hoveniersberg 24, 9000 Gent).
+
+Vanaf **16:00** kunnen jullie elkaars Nexus vernietigen in League of Legends, een Ancient ten val brengen in Dota 2, bommen ontmantelen in Counter Strike: Global Offensive, op ontzagwekkende snelheden door de maps van Trackmania rijden, en nog veel, veel meer.
+
+# Inschrijven is verplicht!
```

Comment:
Misschien kunnen we in de tekst er bij zetten dat inschrijven niet verplicht is, maar als ze het toch doen, dat ze een drankje ofzo kunnen krijgen? Zo was het in 2014